src/HasManyWithInverseRelationship.php

```diff
@@ -8,15 +8,23 @@
 
 class HasManyWithInverseRelationship extends HasMany
 {
-    protected string $relationToParent;
+    /**
+     * @var string
+     */
+    protected $relationToParent;
```

Comment:
```suggestion
    /** @var string */
    protected $relationToParent;
```

I prefer single-line docblocks.